index.html

```diff
@@ -0,0 +1,205 @@
+  <div class="background">
+
+
+    <div class="backgrounMenu">
+    <ul class="backgrounList">
+      <li><a href="#"><img src="images/facebook.png" alt="facebook-icon" ></a></li>
+      <li><a href="#"> <img id="tweeter" src="images/twitter (2).png"   alt="tweeter-icon"></a></li>
+      <li><a href="#">English</a></li>
+      <li><a href="#">My page</a></li>
+      <li><a href="#">logout</a></li>
+    </ul>
+    </div>
+  </div>
+
+
+<!-- the header -->
+
+
+  <header class="header">
+
+
+   <div class="menu">
+   
+    <div id="menuBar"><img src="images/menu.png" 
+      alt="menubar-image" ></div>
+     
+      <div class="logo"><a href="https://nessrine88.github.io/Capstone-Project/"><img src="images/White And Black Modern Abstract Beauty Logo.png"  alt="logo"> </a></div>
+      <div id="menuList">
+        <ul>
+          <li><a href="./about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li><a href="#"><span class="sp">Best offers</span></a></li>
+        </ul>
+      </div>
+    </div>
+    
+  </header>
+<main>
+  <section class="section1">
+  <h2></h2>
+  <h1 class="clipped-text"><span class="h1span">"Take care of your Beauty!"</span> <br> Welcome to Sultan-Oils Company</h1>
+  <p class="paraSection1">Discover the wonders of prickly pear seeds oil at our leading company. Experience the luxurious benefits of our premium skincare and wellness products enriched with essential fatty acids, antioxidants, and vitamins. We prioritize sustainability and ethical sourcing, making a positive impact on both your skin and the environment..</p>
+
+  <div class="event">25 July 2023</div>
+  <div >@Prickly Pear Seeds Oil Beauty Showcase in Tunisia: <br> the Essence of Natural Beauty</div>
+</section>
+<section class="section2">
+  
+  <h3>Main Activities</h3>
+  <div class="line"></div>
+  <div class="programContainer">
+
+    <div class="prog c1">
+      <img src="images/workshop.png" alt="" class="icon">
+      <h2>Workshops and Demonstrations</h2>
+     <a href="#seemore">SEE MORE</a>
+    </div>
+
+    <div class="prog c2">
+      <img src="images/spa.png" alt="" class="icon">
+      <h2>Spa and Relaxation</h2>
+     <a href="#seemore">SEE MORE</a>
+    </div>
+
+    <div class="prog c3">
+      <img src="images/wellness.png" alt="" class="icon">
+      <h2>Wellness Activities</h2>
+     <a href="#seemore">SEE MORE</a>
+    </div>
+
+    <div class="prog c4">
+      <img src="images/showcase.png" alt="" class="icon">
+      <h2>Product Showcase</h2>
+     <a href="#seemore">SEE MORE</a>
+    </div>
+
+    <div class="prog c5">
+      <img src="images/pro1.png" alt="" class="icon">
+      <h2>Expert Panel and Q&A Sessions</h2>
+     <a href="#seemore">SEE MORE</a>
+    </div>
+  </div>
+  <div class="joinUs "><a href="">Reserve Your Spot Now</a> </div>
+  <div class="lecture"></div>
```

Comment:
This `reserve your spot` button should be similar in design to what we have in our design 

Your design :

![image](https://github.com/Nessrine88/Capstone-Project/assets/20537712/1e9486aa-0de1-4209-8c9b-64d895a02ebd)

Design given :
![image](https://github.com/Nessrine88/Capstone-Project/assets/20537712/ef2c619d-c53c-4874-8b65-c2e5d82873da)